crates/languages/src/python.rs

```diff
@@ -1168,31 +893,50 @@ impl ToolchainLister for PythonToolchainProvider {
 
         match toolchain.kind {
             Some(PythonEnvironmentKind::Conda) => {
+                // Honor conda manager preference via env var ZED_CONDA_MANAGER if set:
+                // accepts "conda", "mamba", or "micromamba". Otherwise, use the detected
+                // manager executable if available, falling back to "conda".
+                let manager_cmd = std::env::var("ZED_CONDA_MANAGER")
```

Comment:
That is instead of doing this env var stuff here, we should call `TerminalSettings::get` here and read out the new setting directly.